docs/sources/developers/plugins/extend-the-grafana-ui-with-links-and-commands.md

```diff
@@ -0,0 +1,106 @@
+# Extend the Grafana UI with links and commands
+
+Use the Plugin extensions API with your Grafana App plugins to add links to the Grafana UI. Doing so allows you to direct users to your plugins pages from various "placements" within the Grafana application.
+
+For a plugin to successfully register links it must:
+
+- Be an App plugin.
+- Be preloaded.
```

Comment:
This needs explanation I think (how to preload a plugin)

Reply:
I guess we can link this documentation https://grafana.com/docs/grafana/latest/developers/plugins/metadata/